### PR TITLE
Add license warnings and errors

### PR DESCRIPTION
**Pull request by nkolev92**
Fixes https://github.com/NuGet/Home/issues/7509

### CODE REVIEW

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/errors-and-warnings/NU5030.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5030.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5030?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5031.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5031.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5031?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5032.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5032.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5032?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5033.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5033.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5033?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5034.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5034.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5034?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5035.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5035.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5035?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5124.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5124.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5124?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5125.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5125.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5125?branch=pr-en-us-1173) |
[docs/reference/msbuild-targets.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/msbuild-targets?branch=pr-en-us-1173) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811192318002963-1173%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=TxtzC0EfFNSb7AZNEv358hLFyFz2xBuA%2FaqEl1q1i5c%3D&st=2018-11-19T23%3A14%3A01Z&se=2018-12-20T23%3A19%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/errors-and-warnings/NU5035.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+f1_keywords: 
+  - NU5035
+---
+
+# NuGet Error NU5035
+<pre>The PackageLicenseUrl cannot be used cannot be used in conjunction with the PackageLicenseFile and PackageLicenseExpression.</pre>
+
+### Issue
+
+When using PackageLicenseFile, PackageLicenseExpression or directly the license element in the nuspec, you should not use the licenseUrl metadata. It will be auto-generated by the client tools to a down-level friendly url.
```
Comment (zivkan) [inline docs/reference/errors-and-warnings/NU5035.md]: > When using PackageLicenseFile, PackageLicenseExpression or directly the license element in the nuspec

The bit after the word 'or' needs to be improved/fixed.

File: docs/reference/msbuild-targets.md [hunk 2]
```diff
@@ -50,7 +50,9 @@ Note that the `Owners` and `Summary` properties from `.nuspec` are not supported
 | Description | Description | "Package Description" | |
 | Copyright | Copyright | empty | |
 | RequireLicenseAcceptance | PackageRequireLicenseAcceptance | false | |
-| LicenseUrl | PackageLicenseUrl | empty | |
+| license | PackageLicenseExpression | empty | Corresponds to <license type="expression"> |
+| license | PackageLicenseFile | empty | Corresponds to <license type="file">. You may need to explicitly pack the referenced license file. |
```
Comment (zivkan) [inline docs/reference/msbuild-targets.md]: check the preview link in Karan's comment. The license XML example gets treated as an unknown HTML tag, and is therefore not rendered. You probably need to HTML escape the < and >, or put it in a code block.

File: docs/reference/errors-and-warnings/NU5035.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+author: nkolev92
+ms.date: 11/15/2018
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords: 
+  - NU5035
+---
+
+# NuGet Error NU5035
+<pre>The PackageLicenseUrl cannot be used cannot be used in conjunction with the PackageLicenseFile and PackageLicenseExpression.</pre>
```
[the remark this answers is not in the document]
Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU5035.md]: Nice catch. 
It was a bug in the first PR (fixed in a later one), but when getting the exact messages, I apparently used the wrong changeset :)

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/errors-and-warnings/NU5030.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5030.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5030?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5031.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5031.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5031?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5032.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5032.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5032?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5033.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5033.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5033?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5034.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5034.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5034?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5035.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5035.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5035?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5124.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5124.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5124?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5125.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5125.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5125?branch=pr-en-us-1173) |
[docs/reference/msbuild-targets.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/msbuild-targets?branch=pr-en-us-1173) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C25%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811252245318103-1173%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2Feo%2B%2FP3q3g46lg25oupPEsqEHmNr3ClquoMBEVkl5rE%3D&st=2018-11-25T22%3A41%3A31Z&se=2018-12-26T22%3A46%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/errors-and-warnings/NU5035.md (hunk 1 above)
Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU5035.md]: I've reworded it. 
It was needlessly verbose earlier.

**Comment by nkolev92** [comment]
🔔 @zivkan Ready for another look. 

//cc @karann-msft Can you please take a look as well so we can merge it?

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/errors-and-warnings/NU5030.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5030.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5030?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5031.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5031.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5031?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5032.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5032.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5032?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5033.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5033.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5033?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5034.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5034.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5034?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5035.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5035.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5035?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5124.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5124.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5124?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5125.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5125.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5125?branch=pr-en-us-1173) |
[docs/reference/msbuild-targets.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/msbuild-targets?branch=pr-en-us-1173) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C25%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811252249390802-1173%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=tv%2Bgd3vsNxZBEJwKfruUEKdjV6ph3eVSPqO6l8oR3go%3D&st=2018-11-25T22%3A45%3A39Z&se=2018-12-26T22%3A50%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/errors-and-warnings/NU5030.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5030.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5030?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5031.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5031.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5031?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5032.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5032.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5032?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5033.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5033.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5033?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5034.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5034.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5034?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5035.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5035.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5035?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5124.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5124.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5124?branch=pr-en-us-1173) |
[docs/reference/errors-and-warnings/NU5125.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/errors-and-warnings/NU5125.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5125?branch=pr-en-us-1173) |
[docs/reference/msbuild-targets.md](https://github.com/nkolev92/docs.microsoft.com-nuget/blob/nkolev92-licenseDocs/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/msbuild-targets?branch=pr-en-us-1173) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C25%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811252330460577-1173%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=UaQoHGYbC1todeJaH1FagdJHjqMJaHga%2B%2FAq%2Fbz11Pc%3D&st=2018-11-25T23%3A26%3A45Z&se=2018-12-26T23%3A31%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**